### DISCUSSION TITLE
Automatic Copying Files in AdonisJS Inertia

### DISCUSSION THREAD

**Discussion by zhiwuang** [adonisjs/core#4445] [Ideas]
While working with the AdonisJS Inertia framework, I encountered a situation where the files located in the resources directory, including pages, css, and app.ts, were not automatically moved to the build directory during the production build process. As a result, I had to perform manual copying of these files.

**Comment by Julien-R44** [comment]
not sure to understand when this problem could occur. without reproduction i am not sure if i can help

**Comment by RomainLanz** [comment]
Hey @zhiwuang! 👋🏻 

In addition to what @Julien-R44 said, ensure you are running your codebase in production mode (with `NODE_ENV=production`). Otherwise, it will not use the correct path to load your assets.

**Reply by zhiwuang** [reply]
Hey, it turns out to be true. Impressive.
